packages/ckeditor5-dev-env/lib/release-tools/changelog/writer-options.js

```diff
@@ -129,6 +128,9 @@ function linkGithubUsers( value ) {
 }
 
 function linkGithubIssues( value, issues = null ) {
+	const packageJson = getPackageJson();
+	const issuesUrl = ( typeof packageJson.bugs === 'object' ) ? packageJson.bugs.url : packageJson.bugs;
```

Comment:
It should somehow throw or log an error if one of the repositories doesn't have the `bugs` property set.